specification/attributes/column_naming_convention.md

```diff
@@ -19,16 +18,21 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
-* Column IDs MUST NOT use abbreviations.
-* Column IDs SHOULD NOT use acronyms.
-* Column IDs MUST be alphanumeric with no special characters.
-* Columns that have an ID and a Name MUST have the Id or Name suffix in the Column ID. Display Name for a Column MAY
-  avoid the Name suffix if it is considered superfluous
+* All columns defined by FOCUS MUST follow the following rules:
+  * Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
+  * Column IDs MUST NOT use abbreviations.
+  * Column IDs SHOULD NOT use acronyms.
+  * Column IDs MUST be alphanumeric with no special characters.
+  * Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the Name suffix if there are no other columns with the same name prefix.
+* Custom columns SHOULD follow the same rules as FOCUS columns listed above.
+* All custom columns MUST be prefixed with an underscore (`_`) to distinguish them from FOCUS columns and avoid conflicts in future releases.
```

Comment:
```suggestion
* All custom columns MUST be prefixed with a consistent `x_` prefix to identify them as external, custom columns and distinguish them from FOCUS columns to avoid conflicts in future releases.
```